### PR TITLE
Release/0.13.0

### PR DESCRIPTION
**Pull request by seesharper**
Made some final adjustments :)

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -2,7 +2,7 @@ $scriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent
 $dotnetScriptPath = "$scriptRoot\build\dotnet-script"
 if (!(Test-Path $dotnetScriptPath)) {
     Write-Host "Downloading dotnet-script..."
-    Invoke-Expression ".\install-dotnet-script.ps1"
+    Invoke-Expression "$scriptRoot\build\install-dotnet-script.ps1"
```
Comment (filipw) [inline build.ps1]: how did it work before?! I swear it did 😊

File: src/Dotnet.Script/Dotnet.Script.csproj [hunk 2]
```diff
@@ -2,8 +2,7 @@
 
   <PropertyGroup>
     <Description>Dotnet CLI tool allowing you to run C# (CSX) scripts.</Description>
-    <VersionPrefix>0.13.0</VersionPrefix>
-    <VersionSuffix>beta</VersionSuffix>
+    <VersionPrefix>0.13.0</VersionPrefix>
```
Comment (filipw) [inline src/Dotnet.Script/Dotnet.Script.csproj]: no beta i good

File: build.ps1 (hunk 1 above)
Comment (seesharper) [reply build.ps1]: `.\dotnet-script\build.ps1` did not work. On failes if the dotnet-script folder does not exists within the build folder.  :)

Comment (filipw) [reply build.ps1]: god help me

File: src/Dotnet.Script/Dotnet.Script.csproj (hunk 2 above)
Comment (filipw) [reply src/Dotnet.Script/Dotnet.Script.csproj]: *is

**Pull request merged**